plugins/volumeFilter_test.go

```diff
@@ -322,3 +321,87 @@ func makeManageSellOffer(price string, amount string) *txnbuild.ManageSellOffer
 		Amount:  amount,
 	}
 }
+
+func TestValidateConfig(t *testing.T) {
+	testCases := []struct {
+		name             string
+		sellBaseCapBase  *float64
+		sellBaseCapQuote *float64
+		mode             volumeFilterMode
```

Comment:
should take action as an input too.

don't need to add more than 1 test case to make sure it fails on an invalid value.
all existing cases should take a combination of `buy` and `sell` action (or maybe all are `sell` but only one success case is `buy` to prove the buy case also works)

Reply:
Makes sense. To make this more efficient (vary across buy and sell for existing tests), I've just done the same loop over modes and actions we've done in other test cases.

Reply:
the test has become more complex with this change, therefore harder to maintain.
there's also too many red and green lines which is increasing the PR overhead.
please revert and do the changes that were requested above with only an addition of 1 test case.